src/plugins/openmeteo/__init__.py

```
@@ -0,0 +1,4 @@
+from .datasource import OpenMeteoDataSource
+from .register import register_plugin
+
+__all__ = ['OpenMeteoDataSource', 'register_plugin']
```

Pylint (3.3.7)

[convention] Final newline missing

(C0304)